assembly/assembly-main/src/assembly/bin/docker.sh

```diff
@@ -121,9 +129,9 @@ init() {
     export CHE_WORKSPACE_STORAGE_CREATE_FOLDERS=false
   fi
 
-  # Ensure that the user "user" has permissions for CHE_HOME and CHE_DATA
-  sudo chown -R user:user ${CHE_HOME}
-  sudo chown -R user:user ${CHE_DATA}
+  # Ensure that the user has permissions for CHE_HOME and CHE_DATA
+  sudo chown -R ${CHE_USER}:docker ${CHE_HOME}
+  sudo chown -R ${CHE_USER}:docker ${CHE_DATA}
```

Comment:
`sudo chown -R ${CHE_USER}:docker ${CHE_DATA}` is already done at line 86. Why do you need to do it again?